### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Gijsjan**
Hi,

Thank you for developing jsx-to-string, I already used it in a project. 

I could not get it to work out of the box, two issues:
- `componentData.name` returned a function, I think because of the es6 classes I use? I added `component.type.name` to the conditional, which seems to work.
- `[].includes` is not defined in my browser (Chrome 46), so I replaced it with `[].indexOf`

I added a test for properties which use the spread operator. Thinking about it, it obviously works, but I wanted to reassure. :)

Regards,

Gijs

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ function jsxToString(component, options) {
       (key) => {
         if (key === 'children' ||
           isDefaultProp(component.type.defaultProps, key, component.props[key]) ||
-          opts.ignoreProps.includes(key)) {
+            opts.ignoreProps.indexOf(key) > -1) {
```
Comment (alansouzati) [inline src/index.js]: I believe there is a spacing issue in this line. Would you mind to remove the two extra characters ?

**Comment by alansouzati** [comment]
Thanks, great contributions. There is only one small issue with an extra spacing, apart from that I believe it is good to go!

**Comment by Gijsjan** [comment]
That should do it!

**Comment by alansouzati** [comment]
Thanks for your contribution

**Pull request merged**